packages/actor-http-fetch/lib/FetchInitPreprocessor.ts

```diff
@@ -17,22 +18,14 @@ export class FetchInitPreprocessor implements IFetchInitPreprocessor {
   }
 
   public async handle(init: RequestInit): Promise<RequestInit> {
-    // Convert body Web stream to Node stream, as node-fetch does not support Web streams
-    let halfDuplex = false;
-    if (init.body && typeof init.body !== 'string' && 'getReader' in <any> init.body) {
-      init.body = <any> ActorHttp.toNodeReadable(<any> init.body);
```

Comment:
Are you certain this is not required anymore?

Reply:
I can test it, but the comment mentions that the workaround was for `node-fetch`, which is not used anymore. I thought the CI also tests it, but I think I misremembered how the CI works here. I will look into it!

Reply:
Undici is now used everywhere indeed. But I don't know if it fully supports Web streams.

Reply:
It should, according to the documentation: https://undici.nodejs.org/#/?id=requestbody

Reply:
But did you test it?

Reply:
Yes, it works on my machine<sup>TM</sup>